triage/model_trainers.py

```diff
@@ -1,44 +1,27 @@
-from sklearn.grid_search import ParameterGrid
+import sklearn
+from sklearn.model_selection import ParameterGrid
+
 from sqlalchemy.orm import sessionmaker
-from results_schema import Model, FeatureImportance
+
 import importlib
 import json
 import logging
 import datetime
 import copy
 import pandas
+import numpy as np
 import warnings
+
 from triage.utils import \
     filename_friendly_hash, \
     retrieve_model_id_from_hash, \
     db_retry
 
+from results_schema import Model, FeatureImportance
 
-def get_feature_importances(model):
-    """
-    Get feature importances (from scikit-learn) of trained model.
-
-    Args:
-        model: Trained model
-
-    Returns:
-        Feature importances, or failing that, None
-    """
-
-    try:
-        return model.feature_importances_
-    except:
-        pass
-    try:
-        # Must be 1D for feature importance plot
-        if len(model.coef_) <= 1:
-            return model.coef_[0]
-        else:
-            return model.coef_
-    except:
-        pass
-    return None
-
+from feature_importances import \
```

Comment:
This needs to be 'from triage.feature_importances', right? I was assuming this is why the Travis builds are failing.